File: python/PROBLEMS/1101.py
import sys

def funcao(x, y):
    sum=0
    for i in range(y, x+1):
        print(i, end=' ')
        sum+=i;
    print(f'Sum={sum}')
    

while True:
    x, y = list(map(int, input().split(' ')))
    
    if y > x:
        x, y = y, x
    if x <= 0 or y <= 0:
        sys.exit()
    else:
        funcao(x,y)

File: js/PROBLEMS/1101.js
var input = require('fs').readFileSync('./dev/stdin', 'utf8')
var lines = input.split('\n')

lines = lines.map(function(item){
    return item.split(' ').map(x => parseInt(x))
})

for (i=0;i<lines.length;i++){
    lines[i].sort((x,y) => x-y)
    x = lines[i][1]
    y = lines[i][0]
    sum=0
    s = ''
    if (x>0 && y>0){
        for (j=y;j<=x;j++){
            sum+=j
            s += j + ' '
        }
        console.log( s + "Sum="+sum+"")
    }

}